WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -70,6 +71,9 @@ public class SimulationApiModel
         [JsonProperty(PropertyName = "Statistics")]
         public SimulationStatistics Statistics { get; set; }
 
+        [JsonProperty(PropertyName = "RateLimits")]
+        public IRateLimitingConfig RateLimits { get; set; }
```

Comment:
this is not in line with the other classes and risks to break the API. `IRateLimitingConfig` belongs to the service logic, and could be modified if that changes, but that should not affect the API response schema. There should be a `RateLimits` class in the Web API project which is translated to/from the `RateLimitingConfig` class in the Services project.